.pullapprove.yml

```
@@ -1333,7 +1333,7 @@ groups:
         - dylhunn # Dylan Hunn
         - filipesilva # Filipe Silva
         - gkalpak # Georgios Kalpakas
-        - jelbourn # Jeremy Elbourn
+        - ~jelbourn # Jeremy Elbourn
```

We already don't request reviews for this group, so we don't need to have it here as well.
```suggestion
        - jelbourn # Jeremy Elbourn
```

---

Done